### ISSUE TITLE
Tests failing

### ISSUE THREAD

**Issue by vdice** [tcnksm/ghr#123]
When `make test` is run, a few of the tests fail due to the test URL returning a 404 when not expected:

```
 $ make test
go get  -d
..
go test -v -parallel=4 ./...
=== RUN   TestRun
=== PAUSE TestRun
=== RUN   TestRun_recreate
--- FAIL: TestRun_recreate (0.67s)
    cli_test.go:71: "ghr -username ghtools -repository github-api-test run-recreate ./testdata" exits 11, want 0

        Failed to create GitHub release page: failed to create a release: POST https://api.github.com/repos/ghtools/github-api-test/releases: 404 Not Found []

...
--- FAIL: TestGHR_CreateRelease (0.39s)
    ghr_test.go:31: CreateRelease failed: failed to create a release: POST https://api.github.com/repos/ghtools/github-api-test/releases: 404 Not Found []
--- FAIL: TestRun (0.51s)
    cli_test.go:24: "ghr -username ghtools -repository github-api-test run ./testdata" exits 11, want 0

        Failed to create GitHub release page: failed to create a release: POST https://api.github.com/repos/ghtools/github-api-test/releases: 404 Not Found []

--- FAIL: TestGHR_CreateReleaseWithExistingRelease (1.15s)
    ghr_test.go:104: #0 CreateRelease failed: failed to create a release: POST https://api.github.com/repos/ghtools/github-api-test/releases: 404 Not Found []
--- FAIL: TestGitHubClient (1.17s)
    github_test.go:69: #0 CreateRelease failed: failed to create a release: POST https://api.github.com/repos/ghtools/github-api-test/releases: 404 Not Found []
FAIL
FAIL	github.com/tcnksm/ghr	2.574s
FAIL
make: *** [test] Error 1
```

https://github.com/ghtools/github-api-test appears to be the repo associated with the test server

**Comment by JehandadK** [comment]
#130 Should address this.

**Comment by Songmu** [comment]
Fixed at #130
